spec/integration/util/windows/user_spec.rb

```diff
@@ -130,7 +131,7 @@ def expect_logon_failure_error(&block)
 
       context "with a correct password" do
         it "should return true even if account restrictions are in place " do
-          error = Puppet::Util::Windows::Error.new('', Puppet::Util::Windows::User::ERROR_ACCOUNT_RESTRICTION)
+          error = Puppet::Util::Windows::Error.new('', Puppet::Util::Windows::User::ERROR_ACCOUNT_N)
```

Comment:
Is this a typo? There doesn't seem to be any need for a change to this line, and this new constant is never defined.

Reply:
Right, this was never intended.